Dependencies/Utility/Utility/intrin_compat.h

```diff
@@ -132,17 +133,20 @@ static inline uint64_t _rdtsc()
 #endif
 
 #ifndef cpuid
-#if defined(_MSC_VER)
+#if (defined _M_IX86 || defined _M_X64 || defined __i386__ || defined __amd64__)
```

Comment:
What is the goal of this condition?

Reply:
Goal was to use dummy implementation on architectures other than x86 and x86_64. (As cpuid is instruction specific to x86) Condition was previously only used only for GCC and clang, but same applies to MSVC.